tests/ui/lint/unused/issue-117142-invalid-remove-parens.rs

```diff
@@ -0,0 +1,18 @@
+// check-pass
+#![allow(unused_variables)]
```

Comment:
Isn't this already the default for UI tests (all unused warnings are suppressed)?